src/scss/navbar.scss

```diff
@@ -30,3 +30,7 @@
 .invisible-backing {
   height: $navbar-height;
 }
+
+.avatar-button {
+  cursor: default;
```

Comment:
`cursor: pointer` would be the best fit here since it makes it seem like the avatar is actually clickable